src/lease-details/LeaseDetails.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import {Row, Col} from 'react-bootstrap';
 import moment from 'moment';
+import './lease.css';
```

Comment:
I think once this is imported it will be applied to all col-xs-3 and not only to LeaseDetails.jsx